lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -2744,18 +2805,18 @@ class SqlParser(
             "unpivot" -> {
                 val actualChild = rem.tail.parseExpression(precedence)
                 ParseNode(
-                    UNPIVOT,
-                    rem.head,
-                    listOf(actualChild),
-                    actualChild.remaining
+                        ParseType.UNPIVOT,
+                        rem.head,
+                        listOf(actualChild),
+                        actualChild.remaining
```

Comment:
nit (may be fixed by ktlint in the future): extra whitespace

Reply:
Fixed.